### ISSUE TITLE
Error when using `-var-file` with cloud backend

### ISSUE THREAD

**Issue by riccardopedrielli** [hashicorp/terraform#30204]
### Terraform Version

```
Terraform v1.1.1
on linux_amd64
```

### Terraform Configuration Files

`main.tf`
```terraform
terraform {
  required_version = "~> 1.1"
  cloud {
    organization = "MyOrg"
    workspaces {
      name = "tfvars-issue"
    }
  }
}

provider "null" {
}

variable "mystring" {
  type = string
}
```

`test1.tfvars`
```terraform
mystring = "value"
```

`test2.tfvars`
```terraform
mystring = "\"value\""
```

### Test 1 - Not working

1. `terraform init`
2. `terraform apply -var-file test1.tfvars`

Error:
```
1 error occurred:
	* Invalid HCL in variable "mystring": At 1:12: Unknown token: 1:12 IDENT value
```

### Test 2 - Working

1. `terraform init`
2. `terraform apply -var-file test2.tfvars`

No errors

### Additional Context

It seems that the strings in `.tfvars` files must be _double-double-quoted_ to avoid errors with the Terraform Cloud backend.
The same happens with bool, but in that case a regular double-quote do the trick, since `true` and `false` usually don't need to be quoted.

**Comment by marianocodes** [comment]
Same here. I was able to reproduce this in 1.1.1 and 1.1.2

**Comment by brandonc** [comment]
The fix was released in v1.1.3 but for some reason this issue wasn't closed in response to the PR being merged, and therefore it didn't make the changelog notes. Thank you for the bug report

**Comment by heavenlyren** [comment]
Can confirm :) Thank you
